dataset_statistics.py

```diff
@@ -0,0 +1,50 @@
+import argparse
+import torch
+import torchvision.transforms as transforms
+import torchvision.datasets as datasets
+from tqdm import tqdm
+from PIL import ImageFile
+ImageFile.LOAD_TRUNCATED_IMAGES = True
+
+def main(image_dir: str):
+    
+    """Get mean and std."""
+    dataloader = torch.utils.data.DataLoader(
+        datasets.ImageFolder(root=image_dir, transform=transforms.Compose([
+            transforms.Resize((224, 224)),
+            transforms.ToTensor(),
+        ])), batch_size=4
+    )
+    channels_sum, channels_squared_sum, num_batches = 0, 0, 0
+    count = 0
+    for data, _ in tqdm(dataloader):
+        try:
+            # Mean over batch, height and width, but not over the channels
+            channels_sum += torch.mean(data, dim=[0, 2, 3])
+            channels_squared_sum += torch.mean(data ** 2, dim=[0, 2, 3])
```

Comment:
remove this try except block (see comment in previous code review.) Catching exceptions, ignoring them and continuing is very bad practice!